addons/tmf_loadouts/loadouts/cdf_army_template.hpp

```diff
@@ -0,0 +1,531 @@
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {DOUBLES(CUP_H_CDF_H_PASGT,__CAMO)};
+	primaryWeapon[] = {"CUP_arifle_AK74"};
+	attachment[] = {"cup_acc_flashlight"};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_545x39_AK_M"),
+		LIST_2("CUP_30Rnd_TE1_Green_Tracer_545x39_AK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {DOUBLES(CUP_V_CDF_6B3_4,__CAMO)};
+	primaryWeapon[] = {"CUP_arifle_AK74_GL"};
+	magazines[] += {
+		LIST_4("CUP_1Rnd_HE_GP25_M"),
+		LIST_2("CUP_1Rnd_SMOKE_GP25_M")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	primaryWeapon[] = {"CUP_arifle_AKS74U"};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_SLA_Medicbag"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"B_Carryall_khk"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_vityaz"};
+	magazines[] = {
+		LIST_8("CUP_30Rnd_9x19_Vityaz"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_SmokeRed_GP25_M"),
+		"CUP_IlumFlareRed_GP25_M"
+	};
+	linkedItems[] += {"Binocular","ItemGPS"};
+};
+
+class sl : r {
+	displayName = "Squad Leader";
+	vest[] = {DOUBLES(CUP_V_CDF_6B3_5,__CAMO)};
+	items[] += {"ACE_MapTools"};
+	sidearmWeapon[] = {"CUP_hgun_PMM"};
+	magazines[] += {
+		LIST_2("CUP_12Rnd_9x18_PMM_M"),
+		"SmokeShellGreen"
+	};
+	linkedItems[] += {"Binocular","ItemGPS"};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {DOUBLES(CUP_H_CDF_OfficerCap,__CAMO)};
+};
+
+class fac : g {
+	displayName = "Forward Air Controller";
+	backpack[] = {"CUP_B_Kombat_Radio_Olive"};
+	backpackItems[] =
+	{
+		FAC_GEAR,
+		LIST_2("CUP_1Rnd_SmokeGreen_GP25_M"),
+		LIST_2("CUP_1Rnd_SmokeRed_GP25_M"),
+		LIST_2("CUP_1Rnd_SmokeYellow_GP25_M"),
+		LIST_2("CUP_1Rnd_SMOKE_GP25_M")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"CUP_arifle_RPK74_45"};
+	magazines[] = {
+		LIST_6("CUP_45Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_FieldPack_khk"};
+	backpackItems[] = {
+		LIST_6("CUP_45Rnd_TE4_LRT4_Green_Tracer_545x39_RPK_M"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"Binocular"};
+};
```

Comment:
AR and AAR only have a combined ammunition count of 540 which will limit their effectiveness. Consider adding a backpack to AR and giving him more ammunition?